pyproject.toml

```
@@ -0,0 +1,26 @@
+# Copyright (C) 2025, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
+[project]
+name = "setup_ovs"
+version = "1.0"
+description = "Apply an OVS configuration from a JSON file"
+authors = [
+    { name = "RTE", email = "mathieu.dupre@savoirfairelinux.com" }
+]
+license = { text = "Apache License 2.0" }
```

```suggestion
license = { text = "Apache-2.0" }
```
As far as I understood, the short SPDX identifier must be used here: 
* https://packaging.python.org/en/latest/specifications/pyproject-toml/#license
* https://peps.python.org/pep-0639/#spdx-license-expression-syntax